---
title: Applying neural networks to the PiggyBac transposon screen
author: theo
type: post
date: -001-11-30T00:00:00+00:00
draft: true
url: /?p=168
categories:
  - Uncategorized

---
Earlier this year a tour-de-force in _Plasmodium _genetics emerged. Researchers from the Adams lab at the University of South Florida, had succeeding in using the piggyBac transposable element to screen the _P. falciparum _genome to the point of saturation. This is the first ever genome-scale forward genetic screen in _Plasmodium__. _(I participated in a large-scale reverse-genetic screen in _P. berghei_ that was published last year).

It is a truly extraordinary achievement, and a fantastic resource for the malaria community. Importantly, the researchers made available their full dataset &#8211; not just their own phenotypes but the locations of each PiggyBac insertion and sequencing depth. This allows the research community to undertake iterative reanalyses of the underlying data, and it is such an analysis that I discuss here.

The paper provides two values for each gene in the _P. falciparum_ genome, a mutant fitness score (MFS) and a mutant insertion score (MIS). The MIS is described as the best evidence for  arguably the most important phenotype feature &#8211; the classification of whether a gene is essential or dispensable.

It definitely is a good indication of this property for most genes. For instance the authors plot a ROC curve, comparing _Essential_ calls in _P. berghei _and _T. gondii_ with the MIS score:

<img class="alignnone wp-image-170" src="/post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-8.48.05-PM.png" alt="" width="467" height="403" srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-8.48.05-PM.png 1142w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-8.48.05-PM-300x259.png 300w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-8.48.05-PM-768x663.png 768w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-8.48.05-PM-1024x884.png 1024w" sizes="(max-width: 467px) 85vw, 467px" /> 

If the MIS had no relation to phenotype in these other species this graph would follow the diagonal blackline. As it is the area under the curve is substantially above, reflecting the fact that MIS is predictive of phenotype in _P. berghei_. If MIS could perfectly predict phenotype in _P. berghei _this graph would be shaped such that the green line went straight up to the top-left corner and then across. The fact that it doesn&#8217;t could reflect three things:

<ol type="A">
  <li>
    true biological differences in phenotype between <em>P. berghei </em>and <em>P. falciparum</em>
  </li>
  <li>
    technical issues leading to incorrect measurements in <em>P. berghei</em> for some genes
  </li>
  <li>
    technical issues leading to incorrect measurements in <em>P. berghei</em> for some genes
  </li>
</ol>

In actual fact, it is certainly a combination of all three effects. My own sense is that explanation A is relatively rare.  I know that explanation B definitely occurs. There are some genes which are known to be dispensable which we in our screen call as essential (I do not believe there are cases that go the opposite way). The most likely explanation is that the vectors we made to target these genes had mutations that rendered them unable to integrate, or unable to confer resistance.

At one point when I was adding the data to [PhenoPlasm][1], I became concerned about explanation C. I had initially misentered the data by coding Dispensable and Essential from a column in the supplementary data which said either _Mutable_ or _Non-mutable_. When I examined the Rifin genes in this way the vast majority were Essential. This was a surprise, since this is a large multi-gene family and the expectation is that individual members are likely to be dispensable. When I  changed to an assignment based on the MIS, these results were improved, but this was counter-intuitive since I noticed that a number of these now _Dispensable_ genes had not actually had any PiggyBac insertions in their CDS. To understand the reason for this I dived into the methods which explains how the MIS score is computed.

> We calculated the initial score MIS of gene MS<sub>g</sub> based on the equation
> 
><img class="alignnone wp-image-172" src="/post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-9.23.36-PM.png" alt="" width="327" height="144" srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-9.23.36-PM.png 640w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/08/Screen-Shot-2018-08-12-at-9.23.36-PM-300x132.png 300w" sizes="(max-width: 327px) 85vw, 327px" /> 
> 
> where N<sub>g</sub> represents the number of insertions on gene g. D<sub>g</sub> is the TTAA density of the gene g, which could be represented as the number of TTAA per kb of the CDS.

This formula strikes me as slightly odd. It divides the number of insertions in the gene by the density of TTAA sites in the gene. The units of Ng are &#8220;DNA sites&#8221;, and the units of Dg are &#8220;DNA sites per kb&#8221;.  I don&#8217;t see why Dg is corrected for gene length if Ng is not. The effect is that as a gene gets longer, its MIS value gets larger, even if the number of TTAA sites, and the proportion of them with piggyBac insertions, remains the same.

The authors then do some clever maths that argues that the resultant distribution across MSg must actually represent a mixture of two distributions, one for genes which are dispensable and one for genes which are essential. The maths there is a bit beyond me, but the MIS ends up as the posterior distribution to be dispensable. One thing that confuses me about this is that there are quite a few cases of genes with an MIS of 1, i.e. &#8216;certainty of dispensability&#8217; with no insertions into their CDS.

I decided to apply a neural network based approach to this problem. I would train a network to predict _P. berghei_ phenotype from _P. falciparum_ data. It&#8217;s important to note that this doesn&#8217;t rely on _P. berghei_ data always being the right answer for _P. falciparum_. It relies only on the idea that there is some relationship between these two results, which we do expect there is.

&nbsp;

&nbsp;

&nbsp;

 [1]: http://phenoplasm.org/